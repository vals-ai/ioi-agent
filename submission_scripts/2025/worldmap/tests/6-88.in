version https://git-lfs.github.com/spec/v1
oid sha256:75cacc2e7fb4ddb157d7548c75acf71667479565286012c53274316d98441fd2
size 63124
