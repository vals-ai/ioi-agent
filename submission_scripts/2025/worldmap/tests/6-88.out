version https://git-lfs.github.com/spec/v1
oid sha256:9438f950874e86ebd24f274a5e46c740a25585d383fef7898db86076f0acfe91
size 770834
